clang/test/Modules/Inputs/feature-availability/module.modulemap

```diff
@@ -0,0 +1,9 @@
+module Feature1 {
+  header "feature1.h"
+  export *
+}
+
+module Feature2 {
+  header "feature2.h"
```

Comment:
It might also be a good idea to test having a transitively imported module that defines a feature but is not re-exported by the module that imports it and therefore lookup of the feature should fail in the client